content/manual-references.json

```diff
@@ -147,5 +147,44 @@
     "publisher": "Institute of Electrical and Electronics Engineers (IEEE)",
     "DOI": "10.1109/bibm.2018.8621568",
     "URL": "https://doi.org/ggzb26"
+  },
+  {
+    "author": [
+      {
+        "family": "Bordes",
+        "given": "Antoine"
+      },
+      {
+        "family": "Usunier",
+        "given": "Nicolas"
+      },
+      {
+        "family": "Garcia-Durán",
+        "given": "Alberto"
+      },
+      {
+        "family": "Weston",
+        "given": "Jason"
+      },
+      {
+        "family": "Yakhnenko",
+        "given": "Oksana"
+      }
+    ],
+    "collection-title": "NIPS’13",
+    "container-title": "Proceedings of the 26th international conference on neural information processing systems - volume 2",
+    "id": "10.5555/2999792.2999923",
```

Comment:
don't you love when publishers act like they've registered a DOI but haven't actually.


we can change the `id` to `transe` and cite it like `@transe`. Manubot no longer needs raw prefix.

also let's include the URL as https://dl.acm.org/doi/10.5555/2999792.2999923